harvard-lts/jstor_aspace_transformer
# Ss 262

**mongo datestamps.**
* * *

**JIRA Ticket**: [(link)](https://jira.huit.harvard.edu/browse/SS-253)

**Other links**: [(link)](: https://jira.huit.harvard.edu/browse/SS-262)

# What does this Pull Request do?
- fix datestamps for last_update mongo field to now rather than 0 time; maintain harvest dates
TO DO:
Larger Context should be getting picked up, working locally but not in dev; this pr doesn't regress anything, so for now it just fixes datestamps, leaving in non-breaking code for largercontext until I figure out what the problem is; then we will have this to test
- After writing to filesys, integrate larger context (group) recs with work recs for processing/reporting


# How should this be tested?
Deploy to dev
Add group rec to LARGERCONTEXT for testing
run itest
Check timestamps on last_update mongo fields, should be now time, not 0


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
